source/dub/internal/vibecompat/inet/path.d

```diff
@@ -17,21 +18,28 @@ import std.exception;
 import std.string;
 
 
+deprecated("Use NativePath instead.")
```

Comment:
Maybe it's a good idea to point out when or in which version the deprecation will be removed?